website/docs/reference/warehouse-profiles/layer-profile.md

```diff
@@ -2,100 +2,160 @@
 title: "Layer Profile"
 ---
 
-
 ## Overview of dbt-layer
-**Maintained by:** Layer      
-**Authors:** Mehmet Ecevit  
-**Source:** [Github](https://github.com/layerai/dbt-layer)  
-**Core version:** `v1.0` and newer      
-**dbt Cloud:** Not Supported      
-**dbt Slack channel** [#db-layer](https://slack.com)
 
-[![dbt-synapse stars](https://img.shields.io/github/stars/layerai/dbt-adapters?style=for-the-badge)](https://github.com/layerai/dbt-adapters)
-[![latest version on PyPI](https://img.shields.io/pypi/v/dbt-layer-bigquery?style=for-the-badge)](https://pypi.org/project/dbt-layer-bigquery)
+**Maintained by:** [Layer](https://layer.ai)
+**Authors:** [Layer](https://layer.ai)
+**Source:** [Github](https://github.com/layerai/dbt-layer)
+**Core version:** `v1.2` and newer
+**dbt Cloud:** Not Supported
+**dbt Slack channel** [#tools-layer](https://slack.com)
+
+[![dbt-layer stars](https://img.shields.io/github/stars/layerai/dbt-adapters?style=for-the-badge)](https://github.com/layerai/dbt-layer)
+[![latest version on PyPI](https://img.shields.io/pypi/v/dbt-layer?style=for-the-badge)](https://pypi.org/project/dbt-layer)
 
-To immediately start using Layer inside your dbt DAG, install the Layer dbt Adapter for BigQuery.
+To start using Layer inside your dbt DAG, get a Layer API key and install the Layer dbt Adapter for your warehouse:
```

Comment:
I think it's worth more clearly saying before you get into technical configuration, that this adapter is supplementary to dbt-bigquery or dbt-snowflake, and maybe more about how it works